installer/packer/scripts/systemd/admiral/admiral.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Admiral is a highly scalable and very lightweight Container Management platform for deploying and managing container based applications.
+Documentation=https://github.com/vmware/admiral
+After=docker.service systemd-networkd.service systemd-resolved.service
+Requires=docker.service
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=5
+ExecStart=/usr/local/bin/docker run -d -p 8282:8282 --name vic-admiral docker.io/vmware/admiral:dev
```

Comment:
I've noticed I've commented on an old commit, still I'd rather prefer to remove the container on `ExecPreStart` rather than as part of the stop.